storage/sql_test.go

```diff
@@ -0,0 +1,423 @@
+				datatype: "text",
+				nullable: false,
+			},
+		},
+	},
+	{
+		name: "miner_sector_infos",
+		columns: []column{
+			{
+				name:     "miner_id",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "sector_id",
+				datatype: "bigint",
+				nullable: false,
+			},
+			{
+				name:     "state_root",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "sealed_cid",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "activation_epoch",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "expiration_epoch",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "deal_weight",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "verified_deal_weight",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "initial_pledge",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "expected_day_reward",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "expected_storage_pledge",
+				datatype: "text",
+				nullable: false,
+			},
+		},
+	},
+
+	{
+		name: "miner_pre_commit_infos",
+		columns: []column{
+			{
+				name:     "miner_id",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "sector_id",
+				datatype: "bigint",
+				nullable: false,
+			},
+			{
+				name:     "state_root",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "sealed_cid",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "seal_rand_epoch",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "expiration_epoch",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "pre_commit_deposit",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "pre_commit_epoch",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "deal_weight",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "verified_deal_weight",
+				datatype: "text",
+				nullable: false,
+			},
+			{
+				name:     "is_replace_capacity",
+				datatype: "boolean",
+				nullable: true,
+			},
+			{
+				name:     "replace_sector_deadline",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "replace_sector_partition",
+				datatype: "bigint",
+				nullable: true,
+			},
+			{
+				name:     "replace_sector_number",
+				datatype: "bigint",
+				nullable: true,
+			},
+		},
+	},
+}
+
+func TestCreateSchema(t *testing.T) {
+	if testing.Short() {
+		t.Skip("short testing specified")
+	}
+
+	db, err := NewDatabase(context.Background(), "postgres://postgres@localhost:5432/postgres?sslmode=disable")
+	if !assert.NoError(t, err, "connecting to database") {
+		return
+	}
+
+	err = resetSchema(db)
+	if !assert.NoError(t, err, "resetting schema") {
+		return
+	}
+
+	err = db.CreateSchema()
+	if !assert.NoError(t, err, "creating schema") {
+		return
+	}
```

Comment:
Is there a reason to use this `if assert.NoError(t,err) { return }` pattern instead of `require.NoError(t,err)`?

Reply:
No, I just forgot all about the require package :smile: I'll change before I commit